pkg/diag/colors/colors.go

```diff
@@ -36,6 +36,38 @@ func Command(s string) string {
 	return colorLeft + s + colorRight
 }
 
+// TrimPartialCommand returns the input string with any partial colorization command trimmed off of the right end of
+// the string.
+func TrimPartialCommand(s string) string {
```

Comment:
I am happy to trust that you've written this function correctly. But I am highly doubtful that I would be able to make any changes to this function without breaking something. Would you mind adding some unit tests?

Reply:
I actually wrote unit tests for this and forgot to `git add` the file :)